lib/firebase/user.dart

```diff
@@ -30,6 +34,18 @@ enum UserType {
       };
 }
 
+// ignore: constant_identifier_names
+enum UserCollection { users, unregistered_users }
+
+extension GetCollection on UserCollection? {
+  DocumentReference<Map<String, dynamic>> doc([String? path]) {
+    final userCollection = this ?? UserCollection.users;
+    final dbCollection = db.collection(userCollection.name);
+
+    return dbCollection.doc(path);
+  }
+}
```

Comment:
Implementing an enum for the user collections makes the database calls a bit more simple.